src/main/java/pageObjects/StatusPage.java

```diff
@@ -0,0 +1,45 @@
+package pageObjects;
+
+import base.BasePage;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.testng.Assert;
+import webElements.single.$;
+
+public class StatusPage extends BasePage {
+
+    private final $ title = $(By.xpath("//h3[text()='Status Codes']"));
+    private final $ link404 = $(By.linkText("404"));
+    private final $ linkHere = $(By.linkText("here"));
+
+    public StatusPage(WebDriver driver) {
+        super(driver);
+    }
+
+    @Override
+    public void waitPageToLoad() {
+        waitPage(title, this.getClass().getSimpleName());
+    }
+
+    @Override
+    public void verifyPage() {
+
+    }
+
+    public void status404() {
+        link404.click();
+        linkHere.waitForVisibility();
+        Assert.assertTrue(linkHere.isDisplayed());
```

Comment:
estos 2 sin redundsantes wait for visibility espera que sea display. Lo que podrías verificar es un texto

Reply:
Se verificará que el link "here" sea visible, la linea 32 será borrada